src/shared/components/mutationTable/MutationTable.tsx

```diff
@@ -179,6 +179,7 @@ export enum MutationTableColumnType {
     GENE_PANEL,
     SIGNAL,
 }
+type MutationTableColumnTypes = MutationTableColumnType | string;
```

Comment:
Having both `MutationTableColumnType` and `MutationTableColumnTypes` is a bit confusing. Maybe call the new type `ExtendedMutationTableColumnType`?